installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+
+[Optional Step] 
+* Use the following command to run a local registry that will be used as a destination:
+
+[source,terminal]
+----
+$ podman run -dt -p 6000:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true --name local-registry docker.io/library/registry:2
+----
+
+[id="building-image-set-configuration"]
+== Building the image set configuration
+
+The Image Set Configuration is the input file used by oc-mirror to discover which images are necessary to be mirrored.
+
+Below is an example which included an Openshift Release (version 4.13.10), an operator catalog with only 3 operators and some additional images (ubi8 and ubi9):
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+    - name: stable-4.13
+      minVersion: 4.13.10
+      maxVersion: 4.13.10
+    graph: true
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.15
+      packages:
+       - name: aws-load-balancer-operator
+       - name: 3scale-operator
+       - name: node-observability-operator
+  additionalImages: 
+   - name: registry.redhat.io/ubi8/ubi:latest
+   - name: registry.redhat.io/ubi9/ubi@sha256:20f695d2a91352d4eaa25107535126727b5945bff38ed36a3e59590f495046f0
+----
+
+More Image Set Configuration example can be found in the additional resources.
+
+[id="oc-mirror-workflows"]
+== oc-mirror workflows
+
+This section will explain all the workflows supported by oc-mirror currently.
+
+.Mirror to Disk
+
+Pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml <file path> --v2
+----
+
+.Disk to Mirror
+
+Copy the containers images from the tar archive to a container registry (--from flag is required on this workflow).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --from file:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+.Mirror To Mirror
+
+Copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml --workspace ffile:///oc-mirror/mirror1 docker://localhost:6000 --v2
+----
+
+[id="oc-mirror-delete"]
+== Image delete workflow in oc-mirror
+
+In V1, pruning (or deletion of images) is set automatically, if a specific index (release, catalog) or additionalImage is to be deleted, the current workflow would be to use the same "StorageConfig" setting and omit the previously mirrored images, the pruning logic will find the "difference" and prune the images accordingly.
+
+This has its pros and cons, but due to the large amount of bugs been filed, we decided to change the behaviour in v2.
+
+In v2 a new kind called "DeleteImageSetConfiguration" has been introduced, this is used to configure the deletion of images, to avoid using an ImageSetConfig, accidentally deleting wanted (deployed) images.
```

Comment:
```suggestion
In v2 a new kind called "DeleteImageSetConfiguration" has been introduced, this is used to configure the deletion of images, to avoid using an ImageSetConfig, accidentally deleting useful (deployed) images.
```

Reply:
s/wanted/required

Reply:
Ack.